.github/workflows/ci-model-regression.yml

```diff
@@ -418,6 +418,12 @@ jobs:
     if: "!contains(github.event.pull_request.labels.*.name, 'runner:gpu') && github.repository == 'RasaHQ/rasa' && contains(github.event.pull_request.labels.*.name, 'status:model-regression-tests') && needs.read_test_configuration.outputs.configuration_id != ''"
 
     steps:
+      - name: Run DataDog Agent
+        run: |
+          docker run -d --name dd_agent -p 8125:8125/udp -p 8126:8126 -v /var/run/docker.sock:/var/run/docker.sock:ro -v /proc/:/host/proc/:ro -v /sys/fs/cgroup/:/host/sys/fs/cgroup:ro -e DD_API_KEY=${{ secrets.DD_API_KEY }} -e DD_SITE="datadoghq.eu" -e DD_DOGSTATSD_NON_LOCAL_TRAFFIC=true -e "DD_INSIDE_CI=true" -e "DD_HOSTNAME=none" -e GITHUB_ACTIONS=true -e CI=true gcr.io/datadoghq/agent:7
```

Comment:
If we send metrics via API/DataDog client, and we removed ddtrace, using the agent is not needed then.